components/login/index.jsx

```diff
@@ -79,6 +88,15 @@ export default function LoginForm() {
 
   return (
     <div className={bem("login-form")}>
+      {/* FIXME: remove after pod migration is completed */}
+      {shouldDisplayWarning && (
+        <Message variant="warning" prominent hasIcon>
+          Sorry, we cannot log you in at the moment due to planned system
+          maintenance. Please see our
+          <a href="https://inrupt.com/blog/pod-spaces-upgrade"> blog post </a>
+          for more information.
```

Comment:
```suggestion
          maintenance. Please see our{" "}
          <a href="https://inrupt.com/blog/pod-spaces-upgrade">blog post</a>{" "}
          for more information.
```

To fix the link underline:
<img width="386" alt="Screenshot 2022-07-20 at 17 22 52" src="https://user-images.githubusercontent.com/30827/180020755-b61dc103-dbc8-4adc-ad78-2365c43200c1.png">